DNN_Platform.sln

```diff
@@ -536,6 +540,8 @@ Global
 		{3CD5F6B8-8360-4862-80B6-F402892DB7DD}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
 		{3CD5F6B8-8360-4862-80B6-F402892DB7DD}.Debug|Any CPU.Build.0 = Debug|Any CPU
 		{3CD5F6B8-8360-4862-80B6-F402892DB7DD}.Debug|x86.ActiveCfg = Debug|Any CPU
+		{3CD5F6B8-8360-4862-80B6-F402892DB7DD}.Debug-GTM|Any CPU.ActiveCfg = Debug|Any CPU
+		{3CD5F6B8-8360-4862-80B6-F402892DB7DD}.Debug-GTM|x86.ActiveCfg = Debug|Any CPU

```

Comment:
Can the multiple Debug-GTM be removed from the solution

Reply:
Yes, that was me trying to need a full platform build.